<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Booking</class>
 <widget class="QDialog" name="Booking">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Zarezerwuj pojazd</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	font-family: &quot;Helvetica&quot;;
}

QDialog {
background-color: qradialgradient(spread:reflect, cx:0.5, cy:0.5, radius:1.749, fx:0.5, fy:0.5, stop:0 rgba(5, 15, 52, 194), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));
}

QPushButton {
	background-color: rgb(203, 177, 75);
	border: 3px outset rgb(203, 177, 75);
	border-radius: 10px;
	width: 100%;
}
QPushButton:hover {
	background-color: rgb(137, 163, 89);
	border-color: rgb(137, 163, 89);
}
QPushButton:pressed {
	border: 3px inset rgb(137, 163, 89);
}

QGroupBox {
	border: 3px inset gray;
	background-color: none;
	color:rblack;
}

QLabel {
	background-color: none;
	font: 700 12pt;
	color: rgb(203, 177, 75);
}

QLineEdit {
	background-color: white;
	font: 700 11pt;
	border: 3px inset gray;
}</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>700</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 11.5pt;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>661</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 20pt;
text-decoration: underline;
text-transform: uppercase;
color: rgb(159, 144, 69);
background-color: none;</string>
   </property>
   <property name="text">
    <string>Zarezerwuj pojazd</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="person_box">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>510</y>
     <width>771</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:none;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>19</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 11pt;</string>
    </property>
    <property name="text">
     <string>ImiÄ™:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>99</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 11pt;</string>
    </property>
    <property name="text">
     <string>Nazwisko:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>19</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 11pt;</string>
    </property>
    <property name="text">
     <string>E-mail</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>99</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 11pt;</string>
    </property>
    <property name="text">
     <string>Telefon:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>47</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="last_name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>127</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="phone">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>127</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="email">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>47</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>name</zorder>
   <zorder>last_name</zorder>
   <zorder>phone</zorder>
   <zorder>email</zorder>
   <zorder>company_box</zorder>
   <zorder>company_box</zorder>
   <zorder>company_box</zorder>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>221</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:none;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="reg_num">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>99</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 11pt;</string>
    </property>
    <property name="text">
     <string>Numer rejestracyjny:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 11pt ;</string>
    </property>
    <property name="text">
     <string>Wybierz:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="person">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>191</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(157, 158, 98);
font: 700 11pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Osoba prywatna</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="company">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(157, 158, 98);
font: 700 11pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Firma</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="bookVehicles">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>1001</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:none;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="company_box">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>510</y>
     <width>771</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:none;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>19</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 11pt;</string>
    </property>
    <property name="text">
     <string>E-mail</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>99</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 11pt;</string>
    </property>
    <property name="text">
     <string>Telefon:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="company_phone">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>127</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="company_email">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>47</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="company_name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>47</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>19</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 11pt;</string>
    </property>
    <property name="text">
     <string>Nazwa firmy</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Booking</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Booking</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
